tests/engines/registries/acceptance/edit-revision/revision-test.ts

```diff
@@ -403,6 +406,74 @@ module('Registries | Acceptance | registries revision', hooks => {
         assert.dom('[data-test-page-label]').containsText('This is the second page');
     });
 
+    test('it shows failed saves/deletes', async function(this: RevisionTestContext, assert) {
+        server.namespace = '/v2';
+        server.patch('/schema_responses/:id', () => ({
+            errors: [{ detail: 'Fail to save schema response' }],
+        }), 400);
+        server.del('/schema_responses/:id', () => ({
+            errors: [{ detail: 'Fail to delete schema response' }],
+        }), 403);
+        const initiatedBy = server.create('user', 'loggedIn');
+        const revision = server.create(
+            'schema-response', {
+                initiatedBy,
+                revisionResponses: {},
+                registration: this.registration,
+            },
+        );
+
+        await visit(`/registries/revisions/${revision.id}/`);
+        // Fail justification save
+        await triggerKeyEvent('textarea[name="revisionJustification"]', 'keyup', 32);
+        assert.dom('#toast-container', document as any).containsText(
+            t('registries.drafts.draft.metadata.failed_auto_save'),
+            'Error toast on failed metadata save',
+        );
+        // wait for toast to disappear. TODO: find a better way to do this. sinon.useFakeTimers()??
```

Comment:
Same thoughts here. I'd like to not have to wait 10 extra seconds in the tests if we don't have to.